sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/GalleriesOperationsExtensions.cs

```diff
@@ -150,12 +154,16 @@ public static Gallery Get(this IGalleriesOperations operations, string resourceG
             /// <param name='galleryName'>
             /// The name of the Shared Image Gallery.
             /// </param>
+            /// <param name='select'>
+            /// The select expression to apply on the operation. Possible values include:
+            /// 'Permissions'
+            /// </param>
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<Gallery> GetAsync(this IGalleriesOperations operations, string resourceGroupName, string galleryName, CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<Gallery> GetAsync(this IGalleriesOperations operations, string resourceGroupName, string galleryName, string select = default(string), CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Also a breaking change

Reply:
This was removed after the latest changes were added to compute-2020-12